templates/cisco_ios_show_ip_ospf_database_router.textfsm

```diff
@@ -0,0 +1,48 @@
+Value Filldown ROUTER_ID (\d+\.\d+\.\d+\.\d+)
+Value Filldown PROCESS_ID (\d+)
+Value Filldown AREA (\d+\.\d+\.\d+\.\d+|\d+)
+Value Filldown LSA_AGE (\d+)
+Value Filldown LSA_OPTIONS (.*)
+Value Filldown LSA_TYPE (.*)
+Value Filldown LSA_ID (\d+\.\d+\.\d+\.\d+)
+Value Filldown LSA_ADV_ROUTER (\d+\.\d+\.\d+\.\d+)
+Value Filldown LSA_SEQ_NUMBER (\d+)
+Value Filldown LSA_CHECKSUM (\S+)
+Value Filldown LSA_LENGTH (\d+)
+Value Filldown LSA_NUM_LINKS (\d+)
+Value LS_LINK_TYPE ((a Stub Network)|(a Transit Network))
```

Comment:
Do you want to just capture whether it's a stub or transit rather than the whole `a Stub Network`?

Reply:
I could be persuaded either way here, so I will defer to what the community thinks is best. 

Philosophically, I have always thought that a TextFSM template should convert *exactly* what the CLI says into structured output. So on one hand, I think "a Stub Network" and "a Transit Network" should be returned instead of "Stub" or "Transit".

On the other hand, I cannot think of a scenario where a user would be inconvenienced by the fact that this template returns "Stub", "Transit", or "point-to-point" (which I'm adding in a commit momentarily) instead of the full text. This could also reduce future LOCs that would filter out the unnecessary bits of what is returned.

It's an easy change either way, so I would say let me know which way you prefer, and I can modify it accordingly.

Reply:
Opted to go with catching all text after support for P2P network types was added. Resolved via 87a1f37.